src/main/java/frc/robot/commands/balancing/PIDSwitchBalancerCommand.java

```diff
@@ -21,9 +21,6 @@ public class PIDSwitchBalancerCommand extends BaseBalancerCommand {
     private double deltaAngle;
 
     private boolean reachedStation;
-    private boolean passedCenter;
-    private boolean waiting;
```

Comment:
I deleted all these boolean states because they weren't used, is there a plan to use them in the future @rishay-jain or are they vestigial?

Reply:
deleting them should be fine for now, I'll test those methods and add them back if needed